### PR TITLE
First Name Changes + Additions

### PR DESCRIPTION
**Pull request by t-toasted**
## About the PR 
Much like the previous PR with last names, adds in around 200-300 new names to each name category. Regions include China, Japan, Korea, India, Thailand, and others. Removes duplicate names, puts the three lists in alphabetical order for anyone who wants to update them in the future.

## Why's this needed? 
More diversity in names is good. Removing duplicated names is also good for cleaning up. 

## Changelog 

```changelog
(u)Roddy
(+)Nanotrasen has begun hiring from a wider pool of employees. Adds in around 500 new random names. 
```
[QoL] [Internal]

### CODE REVIEW

File: strings/names/first_male.txt [hunk 1]
```diff
@@ -1,743 +1,872 @@
...
 	Montague
 	Monte
 	Monty
+	Morgan
+	Moses
+	Muyang
+	Myles
+	Naji
+	Nasir
+	Nasir
```
Comment (Vaskritaya) [inline strings/names/first_male.txt:614]: "nasir" is duplicated

File: strings/names/first_male.txt [hunk 2]
```diff
@@ -1,743 +1,872 @@
...
 	Lorde
+	Lorenzo
 	Loreto
 	Lou
+	Louis
+	Luca
 	Lucas
+	Luhan
+	Luis
+	Luis
```
Comment (Vaskritaya) [inline strings/names/first_male.txt:551]: luis duplicated

Comment (t-toasted) [reply strings/names/first_male.txt]: removed duplicates again. Error on my part from adding and forgetting to set the additions to ignore duplications

**Comment by pali6** [comment]
Doing the addition of names and sorting in separate commits would have made reviewing easier as it's pretty difficult to see what got added and what just rearranged here.

File: strings/names/first.txt [hunk 3]
```diff
@@ -1,1503 +1,1781 @@
...
+	Jianjun
+	Jianping
+	Jie
 	Jillian
 	Jillie
 	Jim
+	Jimena
+	Jimmy
+	Jing
+	Jing Hua
```
Comment (pali6) [inline strings/names/first.txt:858]: Ditto.

File: strings/names/first.txt [hunk 4]
```diff
@@ -1,1503 +1,1781 @@
...
+	Lesly
 	Lessie
 	Leta
+	Levi
+	Lexi
 	Lexia
 	Lexus
+	Li
+	Li Hua
+	Li Mei
```
Comment (pali6) [inline strings/names/first.txt:1072]: .

**Review by pali6 — COMMENTED** [review]
Some other such names in the rest too.

File: strings/names/first.txt [hunk 5]
```diff
@@ -1,1503 +1,1781 @@
...
+	Andrew
+	Andy
+	Aneko
 	Angel
+	Angela
+	Angelica
+	Angelina
+	Angelo
+	Angie
+	Anh Duong
```
[the remark this answers is not in the document]
Comment (t-toasted) [reply strings/names/first.txt:123]: For these, Hui Fen and others can be one of three things: Hui Fen, Hui-Fen, or Huifen. It’s usually just variations and personal preference in readability

**Comment by t-toasted** [comment]
> Doing the addition of names and sorting in separate commits would have made reviewing easier as it's pretty difficult to see what got added and what just rearranged here.

The first commit in the branch (ignoring the oops one, oops) is sorting and just shoving it into a ‘remove duplications’ tool. Nothing was added there, just removing duplicate names that were already present. Unless you meant something else :waaaa:

**Review by pali6 — APPROVED** [review]

**Pull request merged**